x-pack/plugins/maps/public/connected_components/right_side_controls/layer_control/layer_toc/toc_entry/toc_entry_button/toc_entry_button.tsx

```diff
@@ -116,7 +116,7 @@ export class TOCEntryButton extends Component<Props, State> {
         footnotes.push({
           icon: <EuiIcon color="subdued" type="filter" size="s" />,
           message: i18n.translate('xpack.maps.layer.isUsingSearchMsg', {
-            defaultMessage: 'Results narrowed by search bar',
+            defaultMessage: 'Results narrowed by filter',
```

Comment:
@gchaps Is there a word or phrase used to address the search bar in Kibana? Looking at https://www.elastic.co/guide/en/kibana/7.13/kibana-concepts-analysts.html#kibana-concepts-searching-your-data there does not appear to be one.

Reply:
In docs, we use _search query,_ or **query**, for a request for information about data.  Could the tooltip be this:

Results narrowed by your query and filters.

Reply:
I updated the wording to "Results narrowed by query and filters", just removed "your" to generally match other wording in the Maps app